### PR TITLE
Add snacks and .gitignore

### LINKED ISSUE #8: more snax
- gotta have 'em

### PR DESCRIPTION
**Pull request by IrishPrime**
- Add all the snacks.
- Add `.gitignore` file to ignore system files, swap files, and bran
  muffins, which are not a real snack.
- Resolves #8.

### CODE REVIEW

**Comment by IrishPrime** [comment]
Uncertain if we should try to track all possible snacks, as the list may become difficult to maintain, but gets us better coverage, or if we should really only focus on the snacks which we enjoy. Feedback appreciated.

**Comment by slcrider** [comment]
@IrishPrime - it'd be neat to see if there's a snacks list we could pull down and parse from somewhere, or maybe a web scrubber from a site of yummy snacks. A real lazy Google search shows this wonderful Wiki, the "[List of Snack Foods](https://en.wikipedia.org/wiki/List_of_snack_foods)". If I'm not mistaken, Wikipedia has a REST API you can use - [here's their help page](https://www.mediawiki.org/wiki/API:Main_page).

**Review by slcrider — APPROVED** [review]
That's a lot of snacks!

File: SEARCH_SNACK3.pm [hunk 1]
```diff
@@ -6,27 +6,235 @@ use strict;
...
+		"Fudge",
 		"Fun Dip",
-		"Bo Rounds",
-		"Sweet Potato Fries",
-		"Pocky",
-		"Pocky For Men",
+		"Gansito",
+		"Geplak",
+		"Ginger snaps",
+		"Goatse",
```
Comment (IrishPrime) [inline SEARCH_SNACK3.pm:100]: Important.

Comment (noradrenaline) [reply SEARCH_SNACK3.pm:100]: Is this a snack or more of a meal tho?

**Pull request merged**